docs/apache-airflow/concepts/dags.rst

```diff
@@ -140,28 +140,41 @@ You can also provide an ``.airflowignore`` file inside your ``DAG_FOLDER``, or a
+ - *Manually triggered* via the UI or API
+ - Or *scheduled*, defined by ``schedule_interval`` on the DAG. For example::
 
-DAGs do not *require* a schedule, but it's very common to define one. You define it via the ``schedule_interval`` argument, like this::
 
-    with DAG("my_daily_dag", schedule_interval="@daily"):
-        ...
+    from airflow.models import DAG
 
-The ``schedule_interval`` argument takes any value that is a valid `Crontab <https://en.wikipedia.org/wiki/Cron>`_ schedule value, so you could also do::
 
-    with DAG("my_daily_dag", schedule_interval="0 * * * *"):
+    with DAG("my_dag", schedule_interval="0 0 * * *"):
         ...
 
-.. tip::
 
-    For more information on ``schedule_interval`` values, see :doc:`DAG Run </dag-run>`.
+This DAG will run every day at 00:00, as defined by the cron expression given to ``schedule_interval``. The value of ``schedule_interval`` can take several types:
+
++------------------------------------------+----------------------------------------------------------------------------------------------------------------------+--------------------------------------+
+| Type                                     | When to use                                                                                                          | Example                              |
++==========================================+======================================================================================================================+======================================+
+| ``None``                                 | No schedule, use for manually triggered DAGs                                                                         | ``None``                             |
++------------------------------------------+----------------------------------------------------------------------------------------------------------------------+--------------------------------------+
+| Cron expression (``str``)                | To run at cron-based intervals                                                                                       + ``"0 0 * * *"``                      |
++------------------------------------------+----------------------------------------------------------------------------------------------------------------------+--------------------------------------+
+| Cron preset (``str``)                    | Convenience cron expression for readability                                                                          + ``"@daily"``                         |
++------------------------------------------+----------------------------------------------------------------------------------------------------------------------+--------------------------------------+
+| List of cron expressions/presets         | To run at intervals that cannot be expressed by a single cron expression.                                            + ``["0 3 * * *", "0 0 * * MON,TUE"]`` |
```

Comment:
This should say if the mode is "and" or "or" (as implemented it's "or")

Reply:
I can be both "and" and "or". For example, when given `["0 3 * * MON,TUE", "@daily"]`, each expression is looked up in the cron presets and converted if found. Shall I convert to "and/or"?

Reply:
Updated the example to demonstrate `["@daily", "0 3 * * MON,TUE"]` is a valid schedule_interval.